ngraph_bridge/ngraph_encapsulate_op.h

```diff
@@ -42,6 +42,7 @@ class NGraphEncapsulateOp : public OpKernel {
  private:
   NGraphEncapsulateImpl ng_encap_impl;
   std::mutex m_compute_lock;
+  std::map<int, std::string> session_names; // key: graph_id, value: session name (for session's graph, and Encapsulate usage statistics)
```

Comment:
This map can be made a static map in the utils. Doesnt need to be a private member in the encap. as it ties up graph id, with session name. It is independent of the encap op

Reply:
Add a function with the static map inside it, we dont need a global map.
Something like this
https://github.com/tensorflow/ngraph-bridge/blob/master/ngraph_bridge/ngraph_builder.cc#L323